src/command_modules/azure-cli-sf/HISTORY.rst

```diff
@@ -2,9 +2,12 @@
 
 Release History
 ===============
+<<<<<<< HEAD
+
+1.0.9 (2017-09-14)
```

Comment:
Heading should be `unreleased`.
Also, `<<<<<<< HEAD` should not be there.

Reply:
fixed.